### ISSUE TITLE
Restore r.gillespie () syntax

### ISSUE THREAD

**Issue by hsauro** [sys-bio/roadrunner#244]
Previous version supported the r.gillespie() syntax which was a convenient way to quickly do a stochastic simulation. Recent changes means that users now have to do the following to do a stochastic simulation:

r.setIntegrator('gillespie')
result = r.simulate(0,10)

Recommend adding the ability to use r.gillespie() syntax again.

Herbert

**Comment by hsauro** [comment]
Note this method is documented in the public documentation:

http://sys-bio.github.io/roadrunner/python_docs/api_reference.html?highlight=gillespie#RoadRunner.RoadRunner.gillespie

**Comment by 0u812** [comment]
Example of usage:
```
import tellurium as te
r = te.loada ('''
  J0: A -> D; A*0.05
  A = 100
  D = 0
''')

result = r.gillespie (0, 10)
r.plot (result)
```

**Comment by 0u812** [comment]
Looks like Kiri fixed this in sys-bio/tellurium@9c375073b04e4f22755e9cea3aca283d5296d59c
